README.md

```diff
@@ -98,18 +105,26 @@ leader = Raft.leader(:peer1)
 Once we have the leader we can read and write to our state machine:
 
 ```elixir
-{:ok, :foo, :bar} = KVStore.write(leader, :foo, :bar)
-{:ok, :bar}       = KVStore.read(leader, :foo)
-{:error, :key_not_found} = KVStore.read(leader, :baz)
+{:ok, {:ok, :foo, :bar}} = KVStore.write(leader, :foo, :bar)
```

Comment:
I actually think all of these are a bug. We should only be returning the users result if the command succeeds.

Reply:
Yeah that caught me by surprise the first time

Reply:
This has been fixed in master.